packages/components/src/components/files/FileAttachmentForm.tsx

```diff
@@ -183,30 +193,44 @@ export class FileAttachmentForm extends React.Component<FileAttachmentFormProps,
     };
 
     renderButtons() {
-        const { cancelText, onCancel, submitText } = this.props;
+        const { cancelText, onCancel, submitText, compact } = this.props;
 
-        return (
-            <div className="row top-spacing bottom-spacing">
-                <div className="col-xs-6">
-                    <Button onClick={onCancel} bsStyle="default" title={cancelText}>
-                        {cancelText}
-                    </Button>
-                </div>
-                <div className="col-xs-6">
-                    <div className="pull-right">
-                        <Button
-                            className="file-form-submit-btn"
-                            onClick={this.handleSubmit}
-                            bsStyle="success"
-                            disabled={this.state.attachedFiles.size == 0}
-                            title={submitText}
-                        >
-                            {submitText}
+        if (compact) {
+            return (
+                <Button
+                    className={classNames('file-form-submit-btn', {'file-form-submit-btn--compact': compact})}
+                    onClick={this.handleSubmit}
+                    bsStyle="success"
+                    disabled={this.state.attachedFiles.size == 0}
+                    title={submitText}
+                >
+                    {submitText}
+                </Button>
```

Comment:
This code is almost an exact duplicate of the else below. Should pull it out into a const above the if statement and then just use it in both of these cases.